cassovary-core/src/main/scala/com/twitter/cassovary/graph/TestGraph.scala

```diff
@@ -15,15 +15,16 @@ package com.twitter.cassovary.graph
 
 import com.twitter.cassovary.graph.StoredGraphDir._
 import com.twitter.cassovary.util.{Sampling, BinomialDistribution}
-import scala.collection.mutable
+import java.util.concurrent.ConcurrentLinkedQueue
+import scala.collection.mutable.HashMap
+import scala.collection.JavaConverters._
 import scala.util.Random
-import scala.collection.mutable.ArrayBuffer
 
 /**
  * A simple implementation of a DirectedGraph
  */
 case class TestGraph(nodes: Node*) extends DirectedGraph {
-  val nodeTable = new mutable.HashMap[Int, Node]
+  val nodeTable = new HashMap[Int, Node]
```

Comment:
it is considered best practice to explicitly specify mutable.HashMap and import only scala.collection.mutable (see http://docs.scala-lang.org/overviews/collections/overview.html starting from "A useful convention is...")